.github/workflows/test.yaml

```diff
@@ -0,0 +1,29 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: ["3.6", "3.7", "3.8", "3.9"]
```

Comment:
Sorry, I guess we should remove "3.6" from the matrix due to [this](https://github.com/actions/setup-python/issues/544#issuecomment-1324999753)